editor-components/feature/feature9/feature9.module.scss

```diff
@@ -72,29 +72,33 @@
   width: 100%;
   height: auto;
   padding-left: 50px;
-  
+  padding-bottom: 50px;
+  padding-top: 50px;
   
 }
 .visible {
   opacity: 1;
   transform: translateY(0);
   background-color: transparent;
 }
+#card-0 {
+  opacity: 1; 
+}
 .image {
   margin: auto;
   max-width: 60px;
   max-height: 60px;
   min-width: 40px;
   padding: 10px;
-  color: var(  --text-color-white);//white
+  color: var(  --text-color-white);
 }
 
 .sub-title {
   display: block;
   margin: 0;
   font-size: 32px;
   font-weight: bold;
-  color: var(   --text-color-white);//white
+  color: var(   --text-color-white);
```

Comment:
i guess this is not the correct color variables. maybe it should be var(--composer-font-color-primary).
please check kardelen's discord message in general channel.